src/marqo/tensor_search/api.py

```diff
@@ -191,6 +193,29 @@ def memory():
     return memory_profiler.get_memory_profile()
 
 
+@app.post('/validate/{index_name}')
+@utils.enable_ops_api()
+def schema_validation(index_name: str, settings_object: str):
+    try:
+        validate_settings_object(index_name, settings_object)
```

Comment:
@farshidz Is it necessary to create a pydantic class for this function, instead of formating the response in the API?

Reply:
I don't mind either way. A pydantic class would be nice, but this is fine too